### ISSUE TITLE
GDPR?

### ISSUE THREAD

**Issue by LlZARD** [magestats/ideas#4]
Hi,

I don't want [my contributions](https://magestats.net/contributor/tim-bezhashvyly) to be listed at your website. How about GDPR? Can you delete or anonymize them?

**Comment by magestats-net** [comment]
Hi @LlZARD 

Thank you for your input.

First of all, Magestats only represents what is publicly available on Github.
Second, your user doesn't seem to match with the linked one.
Wild guess, you renamed it ;-)

I will think about a solution on this topic.

**Comment by LlZARD** [comment]
Yes. The user has been renamed. If it will be also renamed at mage stats that will suit me fine.
